app/src/components/misc/InstallExtensionCTA/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from "react";
+import { Row, Space, Typography } from "antd";
+import { Alert } from "antd";
+import UAParser from "ua-parser-js";
+import { supportedBrowserExtensions } from "./supportedBrowserExtensions";
+import LINKS from "config/constants/sub/links";
+import APP_CONSTANTS from "../../../config/constants";
+import { InstallExtensionContent } from "./type";
+import {
+  trackViewAllPlatformsClicked,
+  trackExtensionInstallationButtonClicked,
+} from "modules/analytics/events/common/onboarding/index";
+import "./installExtensionCTA.css";
+
+const HEADING = "Install Browser extension to start modifying network requests";
+const SUBHEADING =
+  "Requestly lets developers Modify Headers, Redirect URLs, Switch Hosts, Delay Network requests easily. Private and secure, works locally on your browser.";
+
+const InstallExtensionCTA: React.FC<InstallExtensionContent> = ({
+  eventPage,
+  supportedBrowsers = null,
+  isUpdateRequired = false,
+  heading = HEADING,
+  subHeading = SUBHEADING,
+}) => {
+  const [reloadPage, setReloadPage] = useState(false);
```

Comment:
Can we rename this state variable name to `showReloadPagePrompt` or `isReloadPagePromptVisible`, we are using this state to show the reload page message.